servers/rendering/renderer_rd/shaders/scene_forward_lights_inc.glsl

```diff
@@ -229,8 +229,11 @@ void light_compute(vec3 N, vec3 L, vec3 V, float A, vec3 light_color, bool is_di
 			// Calculate Fresnel using specular occlusion term from Filament:
 			// https://google.github.io/filament/Filament.html#lighting/occlusion/specularocclusion
 			float f90 = clamp(dot(f0, vec3(50.0 * 0.33)), metallic, 1.0);
+			// Multiscattering
+			vec2 dfg = prefiltered_dfg(roughness, cNdotV).xy;
```

Comment:
I don't think we need to lookup the dfg again, we can just pass in the energy compensation term from earlier into the light shader.